geode-core/src/main/java/org/apache/geode/redis/internal/RegionProvider.java

```diff
@@ -259,6 +307,16 @@ public void createRemoteRegionReferenceLocally(ByteArrayWrapper key, RedisDataTy
 
   private Region<?, ?> getOrCreateRegion0(ByteArrayWrapper key, RedisDataType type,
       ExecutionHandlerContext context, boolean addToMeta) {
+
+    String regionName = key.toString();
+
+    // check if type is hash TODO: remove
```

Comment:
I think we can safely remove this TODO comment.
The special processing for hashing is now handled at the HashExecutor (along with its sub-classes) commands level. It does not need to be handled at the RegionProvider level.